consensus/src/consensus/consensus_proxy.rs

```diff
@@ -119,6 +122,14 @@ impl<N: Network> ConsensusProxy<N> {
                     let current_head_number = blockchain.head().block_number();
 
                     for (hash, block_number) in response.receipts {
+                        if block_number < since_block_height {
```

Comment:
This will work if we have less than max transactions in the chain, but it will be nice to have an efficient way to prove txns from a range of blocks. 
@jsdanielh

Reply:
It works in any case. Txs returned are sorted and are only the most recent, even if more than max are on chain. This filter just filters it down further.

Reply:
Ok